App.tsx

```diff
@@ -0,0 +1,31 @@
+import { StatusBar } from 'expo-status-bar';
+import AppLoading from 'expo-app-loading';
+import {
+	useFonts,
+	Roboto_400Regular,
+	Roboto_500Medium,
+	Roboto_700Bold
+} from '@expo-google-fonts/roboto';
+
+import { RegistrationScreen } from "./src/screens/RegistrationScreen";
```

Comment:
hm deferent commas in import ' and " choose 1 option (configure eslint + prettier)